user/pages/04.Reference/05.object-storage/docs.en.md

```diff
@@ -25,7 +25,7 @@ multipart upload                 | yes       |
 bucket listing                   | yes       |
 input compression                | yes       |
 request signature authentication | yes       |
-bucket/object ACL                | partially*|
+bucket/object ACL                | yes*      |
```

Comment:
I'd rather leave "partially" as is raises fewer expectations.

Reply:
Reset the value to partially, fine for me =)